### PR TITLE
Add support for log category wildcards

### PR DESCRIPTION
**Pull request by cfbarbero**
Closes aws/aws-lambda-dotnet#109

### CODE REVIEW

**Comment by normj** [comment]
Thanks for the pull request. I'll take a look. Can you confirm you are ok with this being pulled in under the Apache 2 license?

**Comment by cfbarbero** [comment]
Yes, I'm fine with that. 

Thanks,
Cris

**Comment by cfbarbero** [comment]
@normj anything i can do to help get this merged?

File: Libraries/src/Amazon.Lambda.Logging.AspNetCore/LambdaLoggerOptions.cs [hunk 1]
```diff
@@ -161,20 +162,52 @@ private static bool TryGetSection(IConfiguration configuration, string key, out
                     throw new InvalidCastException($"Unable to convert level '{minLevelValue}' for category '{category}' to LogLevel.");
                 }
 
-                logLevelsMapping[category] = minLevel;
+                if (category.Contains("*"))
+                {
+                    var wildcardLocation = category.IndexOf('*');
+                    // Wildcards are only supported at the end of a Category name!
+                    if (wildcardLocation != category.Length - 1)
```
Comment (normj) [inline Libraries/src/Amazon.Lambda.Logging.AspNetCore/LambdaLoggerOptions.cs]: We should also check that there is only one '*'.

**Comment by normj** [comment]
Pull request looks good. Just a minor nitpick issue. I'll merge it in after you add that check.

**Comment by cfbarbero** [comment]
Added in the wildcard validation and added some tests for wildcard validation.

**Comment by normj** [comment]
Thanks for the pull request. This change will go out with the next release which should go out either later this week or next week.

**Comment by normj** [comment]
Version 1.1.0 of Amazon.Lambda.Logging.AspNetCore was released with this pull request

**Pull request merged**